PhoneBook/src/test/java/humphrey/jeriah/PhoneBookTest.java

```diff
@@ -0,0 +1,91 @@
+package humphrey.jeriah;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by jeriahhumphrey on 2/7/17.
+ */
+public class PhoneBookTest {
+    PhoneBook newBook;
+    @Before
+    public void setUp() {
+        newBook = new PhoneBook();
+
+    }
+    @Test
+    public void addIncreasesSizeTest(){
+        newBook.addEntry("John", "123-456-7890");
+        newBook.addEntry("Mike", "123-455-7266");
+        int expected=2;
+        int actual = newBook.getMap().size();
+        assertEquals(expected, actual);
+
+    }
+    @Test
+    public void addTestContainsKey(){
+        newBook.addEntry("John", "123-456-7890");
+        newBook.addEntry("Mike", "123-455-7266");
+        assertTrue(newBook.getMap().containsKey("Mike"));
```

Comment:
Here you're kind of testing implementation rather than just behavior. If you confirm that a particular entry is in the phonebook (without explicitly getting and examining the map) then you leave implementation decisions more open while still confirming proper behavior. Try testing that you can get Mike's entry from the phonebook.